packages/client/hmi-client/src/components/workflow/tera-simulation-workflow.vue

```diff
@@ -344,16 +344,16 @@ const testOperation: Operation = {
 const models = computed<Model[]>(() => props.project.assets?.models ?? []);
 const datasets = computed<Dataset[]>(() => props.project.assets?.datasets ?? []);
 
-async function selectModel(node: WorkflowNode, data: { id: string }) {
-	droppedAssetId.value = null;
-	node.state.modelId = data.id;
-
+const refreshModelNode = async (node: WorkflowNode) => {
 	// FIXME: Need additional design to work out exactly what to show. June 2023
```

Comment:
Please remove dated comment

Reply:
The comment is still valid - we still need more designs here to accommodate user-selection and grouping. What we have here is a very brute-forced show everything-under-the-sun approach which is not very user-friendly.